api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/attributes/AttributeValues.java

```diff
@@ -0,0 +1,11 @@
+package edu.cornell.mannlib.vitro.webapp.auth.attributes;
+
+public interface AttributeValues {
```

Comment:
Maybe it would be better to avoid using Suffixes and Prefixes for interfaces and implementations. [Link](https://octoperf.com/blog/2016/10/27/impl-classes-are-evil/). 
I will try to find better names for them.

Reply:
I removed this interface as I don't see any reason to store it and renamed classes with Impl suffixes.